src/templates_front/templator_front.org

```diff
@@ -660,7 +688,7 @@ return '\n'.join(result)
 
    #+begin_src c :tangle prefix_front.h :exports none :noweb yes
 const char* trexio_string_of_error(const trexio_exit_code error);
-void trexio_string_of_error_f(const trexio_exit_code error, char result[<<MAX_STRING_LENGTH()>>]);
+void trexio_string_of_error_f(const trexio_exit_code error, const int32_t size_string, char* const result);
    #+end_src
 
    *IMPORTANT!*
```

Comment:
Please rerun the code below via org-mode to generate an updated decoding function (it currently misses a bunch of exit codes that we have). I cannot do that as I no longer have a working org-mode + Python setup on my laptop,

Reply:
Done

Reply:
@scemama I do not see it as done.

Reply:
I did the wrong code block (there are 2 similar blocks). To make things clearer, I moved the 2nd block of code right after the 1st one, and changed a bit the message in my last commit.